docs/src/main/tut/typeclasses/monad.md

```diff
@@ -112,6 +112,8 @@ statement into the monadic context.
 import cats.implicits._
 
 Monad[List].ifM(List(true, false, true))(List(1, 2), List(3, 4))
+
+Monad[List].ifM(List(true, false, false))(List(1, 2), List(3, 4))
```

Comment:
If we did that I think we could leave out the extra example. I will add parameter names. Do you want me to pull the second example?